asammdf/gui/widgets/main.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, files=None, *args, **kwargs):
         icon.addPixmap(QtGui.QPixmap(":/open.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
         action = QtWidgets.QAction(icon, "Open", menu)
```

Comment:
Hello Nick,

please also mark the shortcut next to the action text.

Thank you for the PR!

Reply:
After I added that line, I was surprised to notice that Qt seems to do this automatically.
![image](https://user-images.githubusercontent.com/20824939/74356897-166a1580-4d8d-11ea-8af4-2730393ef99d.png)

Reply:
I was not aware of this. Nice find. It will probably work with the other shortcuts as well